lib/fathead/apple_discussions/parse.py

```diff
@@ -38,12 +38,12 @@
 {image}\t
 {abstract}\t
 {url}
-""".replace("\n", "").replace("\\n", "")
+""".replace("\n", "").replace("\\n", "").replace(r"\x", "\\x").replace("^M", "");
```

Comment:
@pjhampton I don't believe the windows newline is a literal `^M` in the text.

Perhaps this approach is better? https://stackoverflow.com/questions/4599936/handling-r-n-vs-n-newlines-in-python-on-mac-vs-windows#4601716